pkg/reconciler/pipelinerun/controller.go

```diff
@@ -40,10 +40,6 @@ import (
 	"knative.dev/pkg/tracker"
 )
 
-const (
-	resyncPeriod = 10 * time.Hour
-)
-
```

Comment:
This was stale code?

Reply:
Yeah, I cleaned up it's one use previously but missed this.